dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi/beansxml/fat/tests/CDIConfigTest.java

```diff
@@ -87,7 +86,13 @@ public void testCdi12Warning() throws Exception {
     }
 
     @Test
-    @SkipForRepeat(EERepeatActions.EE10_ID)
+    /**
+     * Initially we were going to output a warning if emptyBeansXMLExplicitBeanArchive was set when using a CDI version less than 4.0.
```

Comment:
I suggest not to add this in the code. You can add the comments to the issue instead.